pkg/lint/lintersdb/manager.go

```diff
@@ -263,6 +263,7 @@ func (m Manager) GetAllSupportedLinterConfigs() []*linter.Config {
 			WithURL("https://github.com/ryancurrah/gomodguard"),
 		linter.NewConfig(golinters.NewGodot()).
 			WithPresets(linter.PresetStyle).
+			WithAutoFix().
```

Comment:
could you add at least one test case to `test/testdata/fix/`, please, to ensure that integration works ok?

Reply:
Missed it, thanks. There is a test now.